cmake/modules/FindNetCDF.cmake

```diff
@@ -1,20 +1,29 @@
-#
-# (C) Copyright 2011 ECMWF.
-# (C) Copyright 2020 NOAA
-# (C) Copyright 2021 Helmholtz-Zentrum Hereon
```

Comment:
Not sure if this is necessary, I can revert this specific change if needed

Reply:
Zach, there is a more recent FindNetCDF.cmake file, in my repo. please make sure that you are using that one

Reply:
alright thanks, I will incorporate this

Reply:
Zach let's keep the copyright lines on the new one as they are. Originally the file was developed for ECMWF and
now I have put my additions from the NOAA side.